src/controllers/expensesController.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-console */
+const expenseService = require('../services/expensesServices.js');
+const { getById: getUserById } = require('../services/usersServices.js');
+
+const get = async (req, res) => {
+  const expenses = await expenseService
+    .getAll(req.query)
+    .then((exps) => exps.map(expenseService.normalize));
+
+  res.send(expenses);
+};
+
+const getOne = (req, res) => {
+  const expense = expenseService.normalize(req.entry);
+
+  res.send(expense);
+};
+
+const create = async (req, res) => {
+  const {
+    userId,
+    spentAt = new Date(),
+    title,
+    amount,
+    category,
+    note,
+  } = req.body;
+
+  const user = await getUserById(userId);
```

Comment:
Error handling is minimal here. Consider adding error handling for the `getUserById` function to manage potential exceptions, such as when the database connection fails.